### DISCUSSION TITLE
Is there a way to rename a found tag, e.g. from "AirTag 1234" to "My Bike #1"?

### DISCUSSION THREAD

**Discussion by eelcohn** [seemoo-lab/AirGuard#169] [Q&A] [answered]
Title says it all: Is there a way to rename a found tag, e.g. from "AirTag 1234" to "My Bike #1"? It would help me identify false alarms, and I haven't found this feature in the app yet.

**Comment by imbissbudenaesthetik** [comment, marked as answer]
It is possible to rename found tags. This is possible in the latest version in three different ways:
1. In the Trackers detect list, reachable from the dashboard/High Risk/Tracker detected you can swipe on each item from right to left to reveal a rename tool
2. The same approach of swiping is possible in the devices overview (only visible in the expert mode)
3. If you are in the individual view of a tracker (the one where the map is visible) you can just tap on the name of the tracker

However: Some trackers (this includes Apple AirTags) change their ID every day which means that we are not able to identify them as the same tracker from the day before. This means that you might get a new notification the next day anyway.

The following trackers change their IDs:
Apple AirTags and all other Apple / FindMy Devices
Samsung SmartTags
Google Find My Device Trackers

The following trackers do not change their IDs:
Tile
Chipolo (if using the Chipolo Network)
PebbleBee (if using the PebbleBee Network)
